src/pushsource/_impl/backend/errata_source/errata_source.py

```diff
@@ -247,6 +251,22 @@ def _push_items_from_container_manifests(self, erratum, docker_file_list):
 
         return out
 
+    def _push_items_from_appliance_image_list(self, appliance_image_list):
+        if not appliance_image_list:
+            return []
```

Comment:
Another way around is to simply return `out` (which should be an empty list when not appliance_image_list)

```
        out = []
        
        if appliance_image_list:
            ...  # Do the for loop and fetch the push_items

        return out
```

Reply:
updated